src/main/java/io/kontur/disasterninja/dto/UnitDto.java

```diff
@@ -0,0 +1,13 @@
+package io.kontur.disasterninja.dto;
```

Comment:
shouldn't it be part of io.kontur.disasterninja.domain package?

Reply:
Changed Unit as dto to Unit as domain class